src/main/java/seedu/sellsavvy/model/order/OrderList.java

```diff
@@ -53,11 +54,28 @@ public void add(Order toAdd) {
      * Removes the first instance of an equivalent order from the list.
      * The order must exist in the list.
      */
```

Comment:
Given that method has been changed, please verify if this header comment is still valid. Otherwise, change accordingly as per method and parameter.

Reply:
Also since remove() has been updated to take in index, test cases and instances referencing this method will require refactoring.